command/src/command.proto

```diff
@@ -131,14 +129,13 @@ message HttpListenerConfig {
     required uint32 request_timeout = 10 [default = 10];
     // wether the listener is actively listening on its socket
     required bool active = 11 [default = false];
+    required CustomHttpAnswers http_answers = 12;
```

Comment:
Could we change the required keyword into optional one. The idea is to be able to deprecate and remove it in the future without a breaking change

Reply:
done!